src/csync/csync_exclude.cpp

```diff
@@ -491,6 +512,42 @@ static QString convertToRegexpSyntax(QString exclude)
     return regex;
 }
 
+static QString extractBnameTrigger(const QString &exclude, bool wildcardsMatchSlash)
+{
+    // We can definitely drop everything to the left of a / - that will never match
+    // any bname.
+    QString pattern = exclude.mid(exclude.lastIndexOf('/') + 1);
```

Comment:
I know it might be premature optimisations....  but:

`midRef`    (since you do another mid later,  and you can do   `return ("*" + pattern);` at the end so you'd save some allocations.